src/renderer/components/subscriptions-tab-ui/subscriptions-tab-ui.js

```diff
@@ -97,8 +107,10 @@ export default defineComponent({
       switch (event.key) {
         case 'r':
         case 'R':
+          console.error(this.dataLimit)
+          console.error(this.videoList.length)
           if (!this.isLoading) {
-            this.$emit('refresh')
+            // this.$emit('refresh')
           }
```

Comment:
Looks like there is some debug code leftover here.

Reply:
Also please use `console.debug` or `console.log` for debugging
Otherwise linter won't catch those